assets/css/style.css

```diff
@@ -117,6 +117,7 @@ body {
     width: 50px;
     height: 50px;
     background-color: #bb86fc;
+    color: #fff;
```

Comment:
Use a variable for this, please. The overall change is fine!

Reply:
I'm not sure why but the variables are not working. Can you please confirm this?

Reply:
@anushbhatia